traffic_ops/traffic_ops_golang/cdn/dnssecrefresh.go

```diff
@@ -274,9 +327,20 @@ func doDNSSECKeyRefresh(tx *sql.Tx, tv trafficvault.TrafficVault) {
 		if updatedAny {
 			if err := tv.PutDNSSECKeys(string(cdnInf.CDNName), keys, tx, context.Background()); err != nil {
 				log.Errorln("refreshing DNSSEC Keys: putting keys into Traffic Vault for cdn '" + string(cdnInf.CDNName) + "': " + err.Error())
+				anyErrors = true
 			}
 		}
 	}
+	api.CreateChangeLogRawTx(api.ApiChange, "Refreshed DNSSEC keys", user, tx)
```

Comment:
Noticing a couple of things here:

1.) If the `anyErrors` is true, then we should probably not create a changelog with the success message right away
2.) If some of the key refresh operations failed, maybe we could add some kind of a counter (or maybe DS names/ CDN names) that is transmitted back to the user (using changelog and the alerts in the response) that tells us which keys failed to refresh.

Reply:
I'll go with counters since failures to refresh should be rare and would require digging into the logs which will contain the failed DS or CDN names.